test/extended/util/test.go

```diff
@@ -373,6 +375,14 @@ func checkSuiteFocuses() {
 		testFocused = serialConformanceTestsFilter.Match(testName)
 	}
 
+	// if some tests are to be skipped, update the regexp
+	if skipTestsStr != "" {
```

Comment:
This piece of the code is here because I was not able to find a better place where to extend the `excludedTestsFilter` when the `--skip-tests` flag is set. I tried both `InitTest()` and `ExecuteTest()`. The first one is called even before the flags are set, so it does not work here (unless I set something like ginkgo.Before(handler)). Surprisingly, the `ExecuteTest` is not the right place either as it must be called even before the flags are set. So this was the only place that worked for me. I admit it is hacky. If anyone can suggest a better place for it, let's put it there. Until that, it is block the system containers CI.